src/DynamoCoreWpf/Controls/ZoomBorder.cs

```diff
@@ -173,6 +178,12 @@ private void child_MouseMove(object sender, MouseEventArgs e)
 
             // Change ZoomBorder's child translation
             Vector v = start - e.GetPosition(this);
+
+            if (v.Length == 0.0)
```

Comment:
maybe use a tolerance ?

Reply:
I was thinking of it but the values I have seen here have been 0,1,2 etc and 0 has been 0 in those cases. So I  think this is good enough.